discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -128,23 +128,23 @@ def set_cluster_name(self, cluster_name):
     def select_installation_disk(self, hosts_with_disk_paths):
         self.api_client.select_installation_disk(self.id, hosts_with_disk_paths)
 
-    def set_ocs(self, ocs_enabled):
-        logging.info(f'Enabling Ocs to:{ocs_enabled} for cluster: {self.id}')
-        self.set_operator('ocs', ocs_enabled)
+    def set_ocs(self, ocs_enabled, properties=None):
+        self.set_operator('ocs', ocs_enabled, properties=properties)
 
-    def set_cnv(self, cnv_enabled):
-        logging.info(f'Enabling CNV to:{cnv_enabled} for cluster: {self.id}')
-        self.set_operator('cnv', cnv_enabled)
-
-    def set_operator(self, operator, enabled):
+    def set_operator(self, operator_name, enabled, properties=None):
```

Comment:
Change to `set_olm_operator`

Reply:
I find "enabled" param a little bit confusing. Maybe this function should be divided into `set_olm_operator` and `remove_olm_operator`?